README.md

```diff
@@ -1,32 +1,95 @@
+- [Back-End Development Trello](https://trello.com/b/034hEV4X/rebuild-black-business-backend-dev)
+
+If you see a task in the Trello that seems like something you'd like to tackle, comment on the task expressing your interest. A project manager will assign the task to you. Tasks are then elaborated on in detail within [GitHub Issues](https://github.com/Rebuild-Black-Business/RBB-Website/issues). The Trello card will have a link to the corresponding issue so you can get started!
+
+### Branching Convetions
+
+When contributing, please branch any new feature off of `master` using the following naming conventions:
+
+`feature/task-description` - Feature branches are for all new feature work. The task description should be descriptive enough to quickly understand the work done in this branch. Please keep your features concise and ensure the work done in this branch directly relates to your task.
+
+`fix/task-description` - Fix branches should be leveraged for bug fixes on existing features
+
+`hotfix/task-description` - Hotfix branches should only ever be leveraged for _critical_ bug fixes that need attention immediately. This would only be use if there is a broken feature or bug on production that needs to be addressed urgently.
+
+Once you're ready to start contributing, here's a quick example of the flow you should follow to create new a new feature branch:
+
+```sh
+# Make sure you're on the master branch
+git checkout master
+
+# Pull most recent changes from master
+git pull origin master
+
+# Chek out new feature branch
+git checkout -b feature/task-description
+```
+
+### Pull Requests
+
+To send your changes for review, open a pull request. If you’ve never opened a pull request before, [read Thanoshan’s article on creating a pull request](https://www.freecodecamp.org/news/how-to-make-your-first-pull-request-on-github-3/) for more information!
 
+## Terminology
+
+- User Types - Dynamic user data for the various types of data used on the site. Currently this includes the following
+  - Business In Need
+  - Black Owned Business
+  - Service Organization
+  - Ally
+
+## 🖥️ Tech Details
+
+**Frontend**
+
+The website is built using [Gatsby](https://www.gatsbyjs.org/docs/), with UI components leveraging [Chakra UI](https://chakra-ui.com/getting-started) with custom theming for rapid and consistent component driven development.
+
+**Data**
+
+Data for the various user types is stored in [Airtable](https://airtable.com/). Each user type has it's own table that the site pulls from and displays.
+
+To get access to this data, please join the [Rebuild Black Business Discord](https://discord.gg/272XMuv) and express your interest in helping to your assigned Team Lead. They'll get you read access to the data in a timely manner.
+
+## Install
+
+**Clone the repository**
+
+```sh
+# clone the repo
+git clone git@github.com:Rebuild-Black-Business/RBB-Website.git
```

Comment:
Requiring people to already have an invite before being able to contribute seems like a blocker for community contributions. Making it as simple as possible for someone to quickly fork/fix/PR a small contribution seems like a win — not everyone will be able/willing to actually join in, but many will do hotfixes as they see them if it doesn't require jumping through lots of hoops.

Reply:
Fair point for sure, we'll maintain the forking structure for pull request to reduce the barrier to entry.